docs/dashboards/quick-start.md

```diff
@@ -0,0 +1,45 @@
+title: Sematext Dashboards Quick Start
+description: Sematext Cloud is a modern monitoring, log management, transaction tracing, and real user monitoring system. Dashboarding is system's powerful feature for cohesive data analysis from multiple data sources such as logs, metrics, and events. 
+
+
+After you get logged into Sematext Cloud at <https://apps.sematext.com> (or <https://apps.eu.sematext.com> if using Sematext Cloud Europe), the first step is to create a Dashboard. A Dashboard is a collection for different types of data. This is where you keep logs, metrics, events, infrastructure data and experience statistics in a single screen for easy correlation and troubleshooting.
+
+For example, if you have a development and a production environment, it might make sense to have one Dashboard for each. You can create as many Dashboards as you want.
+
+## Creating a Dashboard
+
+You create a Dashboard by pressing the **+ Create Dashboard** button in the Monitoring tab.
```

Comment:
Here we used bold for a button name.  Above I suggested we add quotes because there was no bolding there.  Either way works.  We should just try to be consistent.